etc/emqx.conf

```diff
@@ -1985,6 +1985,39 @@ listener.wss.external.send_timeout_close = on
 ## Value: single | multiple
 listener.wss.external.mqtt_piggyback = multiple
 
+##--------------------------------------------------------------------
+## Origins
+##--------------------------------------------------------------------
+## Enable origin check in header for websocket connection
+##
+## Value: true | false (default false)
+listener.ws.external.check_origin_enable = false
+
+## Enable origin check in header for secure websocket connection
+##
+## Value: true | false (default false)
+listener.wss.external.check_origin_enable = false
+
+## Allow origin to be absent in header in websocket connection when check_origin_enable is true
+##
+## Value: true | false (default true)
+listener.ws.external.allow_origin_absence = true
+
+## Allow origin to be absent in header in secure websocket connection  when check_origin_enable is true
+##
+## Value: true | false (default true)
+listener.wss.external.allow_origin_absence = true
+
+## Comma separated list of allowed origin in header for websocket connection
+##
+## Value: http://url eg. local http dashboard url - http://localhost:18083, http://127.0.0.1:18083
+listener.ws.external.check_origins = http://localhost:18083, http://127.0.0.1:18083
+
+## Comma separated list of allowed origin in header for secure websocket connection
+##
+## Value: http://url eg. http://localhost:8084, http://127.0.0.1:8084
+listener.wss.external.check_origins = localhost:8084, 127.0.0.1:8084
+
```

Comment:
The way the configuration items were originally written, the contents of these configuration ws and wss should be placed in their respective areas